frontend/india-open-badminton-explorer/style.css

```
@@ -0,0 +1,1419 @@
+}
+
+:root.light-theme {
+  --badminton-navy: #f8f9fa;
+  --badminton-navy-light: #ffffff;
+  --badminton-text: #2c3e50;
+  --badminton-text-muted: #5d6d7e;
+  --badminton-glass: rgba(255, 255, 255, 0.85);
+  --badminton-border: rgba(20, 90, 50, 0.15);
+}
+
+@keyframes fadeInUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)